migration/migrator/data/course_tables.sql

```diff
@@ -694,7 +694,8 @@ CREATE TABLE public.polls (
     question text NOT NULL,
     status text NOT NULL,
     release_date date NOT NULL,
-    image_path text
+    image_path text,
+    question_type text DEFAULT 'single-response'::text
```

Comment:
Better to use `varchar` here if the string is guaranteed to be under 255 characters, as it allows certain performance optimizations within the DB. I realize there's other fields within this table that are `text`, and they should probably be moved to `varchar` as well, except for "question", but no point adding more debt here.

Reply:
Sounds good.  This field will be < 255 characters yes.
I think we should add a couple other options for this value.